ledger/src/blockstore_db.rs

```diff
@@ -2204,10 +2201,14 @@ pub mod tests {
     {
         pub(crate) fn iterator_cf_raw_key(
```

Comment:
should we mark this as test/dev-context-only-utils?

Reply:
It is not obvious from the diff, but this is within the `mod tests`. I'm like 99% sure this caught me off guard when I reviewed the PR from Tyera that introduced this function 🤣